src/renderer/api/endpoints/helm-charts.api.ts

```diff
@@ -27,7 +27,7 @@ export const helmChartsApi = {
         return Object
           .values(data)
           .reduce((allCharts, repoCharts) => allCharts.concat(Object.values(repoCharts)), [])
-          .map(HelmChart.create);
+          .map((charts: HelmChart[]) => HelmChart.create(charts[0]));
```

Comment:
```suggestion
          .map(([chart]) => HelmChart.create(chart));
```

Reply:
Though why are we only getting the first entry?

Reply:
The list is received in a form of "chart groups", for examples:
```
'apm-server: [
  {
    name: 'apm-server',
    ...
    version: '2.1.7'
  },
  {
    name: 'apm-server',
    ...
    version: '2.1.6'
  }
]
```
So we're choosing only last one (`charts[0]`) to avoid listing same chart many times in a row.

Reply:
So it is guaranteed to be sorted in decreasing version order?

Reply:
Added descending sorting by `version` field.